irods/connection.py

```diff
@@ -196,9 +196,11 @@ def _connect(self):
                 "{}:{}".format(*address))
 
         self.socket = s
+
         main_message = StartupPack(
             (self.account.proxy_user, self.account.proxy_zone),
-            (self.account.client_user, self.account.client_zone)
+            (self.account.client_user, self.account.client_zone),
+            self.pool.client_name
```

Comment:
can we call it `application_name` instead of `client_name`?    and keep that consistent throughout the code and documentation?